aten/src/ATen/native/BlasKernel.cpp

```diff
@@ -180,9 +181,10 @@ void gemv(char trans, int64_t m, int64_t n, scalar_t alpha, scalar_t *a, int64_t
     return;
   }
 
+  using opmath_t = at::opmath_type<scalar_t>;
   if ((trans == 'T') || (trans == 't')) {
```

Comment:
What about the non-transposed case?

Reply:
Thanks for reminder, have solved that too.